samples/nrf9160/nrf_cloud_rest_fota/boards/nrf9160dk_nrf9160_ns_0_14_0.overlay

```diff
@@ -16,3 +16,7 @@
 		nordic,drive-mode = <NRF_DRIVE_H0H1>;
 	};
 };
+
+&mx25r64 {
+	status = "okay";
+};
```

Comment:
This was already added in [34d3e94.](https://github.com/nrfconnect/sdk-nrf/pull/11241/commits/34d3e94ee8a93ab710599bba849c06bbc37e5ffb)